scripts/dependencycache.ps1

```diff
@@ -0,0 +1,26 @@
+Function Step-Main {
+    Param (
+		[string]$Command = "default",
+		[string]$Manager = "yarn"
+    )
+
+    Get-Process {
+        switch ( $Command ) {
+            clear {
+                Remove-Item -Recurse -Force dist
+				Remove-Item -Recurse -Force node_modules
+				switch ($Manager) {
+					yarn {
+						Remove-Item -Recurse -Force $(yarn cache dir)
```

Comment:
```suggestion
						Remove-Item ((yarn cache dir) + "\*") -Recurse -Force
```

We have to delete the items *within* the cache, not the cache dir itself. Taken from my snippets in Diri (yarncache Skyra tag) which are tested and I use every so often.

Reply:
Doesn't matter, Yarn re-creates the directory.